src/services_models/reserves.jl

```diff
@@ -81,9 +96,113 @@ function service_requirement_constraint!(
     return
 end
 
+function service_requirement_constraint!(
+    psi_container::PSIContainer,
+    service::SR,
+    model::ServiceModel{SR, StepwiseCostReserve},
+) where {SR <: PSY.Reserve}
+    parameters = model_has_parameters(psi_container)
+    use_forecast_data = model_uses_forecasts(psi_container)
+    initial_time = model_initial_time(psi_container)
+    @debug initial_time
+    time_steps = model_time_steps(psi_container)
```

Comment:
Nit:  These are unused. Do you need them?

Reply:
I removed those lines other than this one
```julia
    initial_time = model_initial_time(psi_container)
    @debug initial_time
```